app/styles/modules/_settings.scss

```diff
@@ -759,8 +759,11 @@ section.modal-panel {
         /*minimum width required for the button to look good without occupying too much space*/
         /*is also the default computed width on desktop screen*/
         min-width: 100px;
+        overflow: hidden;
         text-align: center;
-        width: 20%;
+        text-overflow: ellipsis;
```

Comment:
can we apply the same styles to try and fix these buttons: 
![image](https://user-images.githubusercontent.com/128755/38627839-ebe4cd80-3d7d-11e8-852b-818ff6bbe265.png)